src/index.css

```diff
@@ -3,5 +3,4 @@ html {
 	background-attachment: fixed;
 }
 
-@import '~typeface-roboto/index.css';
-@import '~typeface-caveat/index.css';
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@400;700&family=Open+Sans&family=Ubuntu+Mono&display=swap');
```

Comment:
Should we install the font locally instead?

Reply:
Read https://web.dev/font-best-practices/#using-self-hosted-fonts

Not sure if we really care

Reply:
Good for me.